src/dotnet/Fable.Core/Import/Node/Querystring.fs

```diff
@@ -0,0 +1,20 @@
+[<AutoOpen>]
```

Comment:
Sure. In that case, what should we export for module globals?

I think we have some differing opinions of `url`, `globals` etc...